src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalGATKReadFilterPluginArgumentCollection.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+
+import java.util.*;
+
+/**
+ * {@link GATKReadFilterPluginArgumentCollection} for optional read filters in the command line. It allows:
+ *
+ * - Provide a list of read filters to apply.
+ * - Disable some and/or all read filters.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public class OptionalGATKReadFilterPluginArgumentCollection extends GATKReadFilterPluginArgumentCollection {
```

Comment:
I think a better name for this would be DefaultGATKReadFilterArugmentCollection.